server/controllers/user.js

```diff
@@ -19,13 +24,68 @@ class UserController {
     }
   };
 
+  getUserViaEmail = async (req, res, next) => {
+    try {
+      const userEmail = req.params.email;
+      const user = await this.userService.getUserViaEmail(req.params.email);
+      if (user == null) {
+        next(ApiError.notFound(`User with email ${userEmail} not found`));
+        return;
+      }
+      res.status(200).json(user);
+    } catch (err) {
+      next(ApiError.internal(`${err}`));
+    }
+  };
+
   createUser = async (req, res, next) => {
     try {
-      const userId = await this.userService.createUser(req.body);
+      
```

Comment:
Looks like extra whitespace here

Reply:
Gonna run the linter! My bad, will remember to do next time!